hyperspy/_components/pes_voigt.py

```diff
@@ -0,0 +1,331 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
```

Comment:
probably needs an update more generally

Reply:
Yes, I can change it in the files used here, but it can be found across the project.